backend/src/main/java/com/google/minicrm/servlets/SettingsServlet.java

```diff
@@ -0,0 +1,83 @@
+package com.google.minicrm.servlets;
+
+
+import com.google.appengine.api.datastore.*;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.minicrm.data.Advertiser;
+import com.google.minicrm.data.Currency;
+import com.google.minicrm.data.Language;
+import com.google.minicrm.data.Settings;
+import com.google.minicrm.interfaces.ClientResponse;
+import com.google.minicrm.utils.UserAuthenticationUtil;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Map;
+
+@WebServlet("/api/settings")
+public final class SettingsServlet extends HttpServlet {
+    @Override
```

Comment:
nit: Javadocs on servlet requests can be really helpful in organizing endpoint requests. See other request javadocs for reference and consistency.